Pipfile

```diff
@@ -26,4 +26,4 @@ idna = ">=2.5,<3"
 
 [requires]
 
-python_version = "3.7"
+python_version = "3.10"
```

Comment:
This is not related to CI, should be fix regardless. Without this running pipenv install will install python 3.7 and "pipenv run" will run python 3.7 instead of current version.

Reply:
I found that this does not work with the compiled drivers. We need to recompile them with python 3.10 if we want to upgrade python version, or remove them from the source.

Reply:
I removed this change. We can upgrade python version later.